apps/smart-contracts/core/test/integration/DepositTradeHelper.test.ts

```diff
@@ -549,4 +569,306 @@ describe('=> DepositTradeHelper', () => {
       core.collateral.permit.reset()
     })
   })
+
+  describe('trade testing', () => {
+    let univ3Factory: UniswapV3Factory
+    let positionManager: NonfungiblePositionManager
+    let governanceWstETHSwapParams: IVault.SingleSwapStruct
+    let governanceWstETHFundingParams: IVault.FundManagementStruct
+    let permitForFunder: IDepositTradeHelper.PermitStruct
+    let permitForRecipient: IDepositTradeHelper.PermitStruct
+    let balancerParamsForDeposit: IDepositTradeHelper.OffChainBalancerParamsStruct
+    let uniswapParamsForTrade: IDepositTradeHelper.OffChainTradeParamsStruct
+    const GOVERNANCE_COLLATERAL_SUPPLY = parseEther('1000')
```

Comment:
```suggestion
    const GOVERNANCE_COLLATERAL_SUPPLY = parseEther('1200')
```
so it's easier to divide by 3